### PR TITLE
WoE: implement Watch Your Step

### PR DESCRIPTION
**Pull request by strib**
Issue: #2907

### CODE REVIEW

File: server/game/cards/06-WoE/WatchYourStep.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+const Card = require('../../Card.js');
+
+class WatchYourStep extends Card {
+    // Play: Choose a house on your opponent's identity card. If they
+    // do not choose this house as their active house during their
+    // next turn, make 2 token creatures and ready them.
```
[the remark this answers is not in the document]
Comment (strib) [reply server/game/cards/06-WoE/WatchYourStep.js:6]: Got it, thanks.  I can't figure out how to get a reference to the new tokens I made in a previous action.  If I use something like `ability.actions.sequential()`, can the second action see the event from the first action?

[the remark this answers is not in the document]
Comment (strib) [reply server/game/cards/06-WoE/WatchYourStep.js:6]: Ok I figured out something (kind of ugly) that works here, please take another look!

[the remark this answers is not in the document]
Comment (strib) [reply server/game/cards/06-WoE/WatchYourStep.js:6]: Thanks but I've tried this extensively and `then` clauses don't work here, they never fire within this `nextRoundEffect` no matter what I do.  This is the only way I was able to make it work.

[the remark this answers is not in the document]
Comment (strib) [reply server/game/cards/06-WoE/WatchYourStep.js:6]: (There is no example in the code of a `then:` within a `nextRoundEffect`.)

[the remark this answers is not in the document]
Comment (romanogit) [reply server/game/cards/06-WoE/WatchYourStep.js:6]: And I suppose it does not work?

[the remark this answers is not in the document]
Comment (strib) [reply server/game/cards/06-WoE/WatchYourStep.js:6]: Correct, it doesn't work.  If I put a 'then:` there, nothing every happens in it, even with `alwaysTriggers: true`.

[the remark this answers is not in the document]
Comment (romanogit) [reply server/game/cards/06-WoE/WatchYourStep.js:6]: See #3225

**Comment by romanogit** [comment]
I used your code and adde the missing property.

**Pull request closed without merging**